src/fermilib/utils/__init__.py

```diff
@@ -30,7 +30,8 @@
 from ._molecular_data import MolecularData, periodic_table
 
 from ._operator_utils import (eigenspectrum, commutator,
-                              count_qubits, is_identity)
+                              count_qubits, is_identity,
+                              save_operator, load_operator, get_file_path)
```

Comment:
can you please make the imports in alphabetical order (i.e. get_file_path, load_operator, save_operator)?